spec/integration/get_more_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'spec_helper'
+
+describe 'getMore operation' do
+  # https://jira.mongodb.org/browse/RUBY-1987
+  min_server_fcv '3.2'
+
+  let(:collection) do
+    subscribed_client['get_more_spec']
+  end
+
+  let(:scope) do
+    collection.find.batch_size(1).each
+  end
+
+  before do
+    collection.delete_many
+    collection.insert_one(a: 1)
+    #collection.insert_one(a: 2)
```

Comment:
Did you mean to leave this comment here?

Reply:
Yes, the second insert can be helpful for ensuring there is enough data.